flocker/cli/functional/test_deploy_script.py

```diff
@@ -85,6 +99,7 @@ def _send_configuration(self,
         deployment_config.setContent(safe_dump(deployment_config_yaml))
         return getProcessOutputAndValue(
             b"flocker-deploy", [
+                b"--certificates-directory", self.certificate_path.path,
                 b"--port", unicode(self.port_number).encode("ascii"),
                 b"localhost", deployment_config.path, app_config.path],
             env=environ)
```

Comment:
This method looks a lot like the acceptance test helper `flocker_deploy`.  File an issue for refactoring.